scss/partials/_agenda.scss

```diff
@@ -0,0 +1,180 @@
+.tab-list {
+    align-items: stretch;
+    display: inline-flex;
+    width: 100%;
+    margin: 10px 0 0 0;
+    border-bottom: solid 1px #ccc;
+
+    li {
+        width: 25%;
+        text-align: center;
+        margin-bottom: -1px;
+        @include border-radius(10% 10% 0 0);
+
+        a {
+          color: $light-purple;
+          padding: 26px .5px;
+          display: block;
+          font-size: 0.75rem;
+          @media (min-width: 1024px) {
+             font-size: 1.25rem;
+          }
+        }
+    }
+
+    .active {
+      border: solid 1px #ccc;
+      border-bottom-color: #fff;
+      a {
+        color: $dark-purple;
+      }
+    }
+}
+
+.timeline-container {
+    margin: 0;
+
+    h3 {
+      text-transform: uppercase;
+      padding-top: 3rem;
+    }
+}
+
+.timeline-item {
+    margin: 0;
+    position: relative;
+
+    li {
+      margin: 48px 0;
+      display: flex;
+      position: relative;
+
+      &:before {
+        content: ' ';
+        width: 18px;
+        height: 18px;
+        left: 77px;
+        top: 35%;
+        position: absolute;
+        background: $light-purple;
+        @include border-radius(50%);
+
+        @include landscape {
+          top: 39%;
+        }
+
+        @include tablet {
+          left: 116px;
+          top: 39%;
+        }
+      }
+    }
+
+    span {
+      margin-right: 60px;
+      display: flex;
+      align-items: center;
+      font-size: rem(20px);
+      font-family: monospace;

```

Comment:
what do you think about putting this font as a variable too?

Reply:
what would be its name?